io.mrarm.mcpelauncher-client.json

```diff
@@ -0,0 +1,113 @@
+{
+    "id": "io.mrarm.mcpelauncher-client",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
```

Comment:
It really should be upgraded. What was the problem with 5.14?

Reply:
With 5.14 the launcher would loose its i386 support, which can run much more versions of Minecraft on Linux (from last years), but specifing 5.12 only for i386 won't seem to work.
I guess only provide mcpelauncher-client32 (i386) in the AppImage not in flatpak, then the new sdk is fine.

Reply:
Old Minecraft Version support for intel / amd.
The x86_64 launcher also needs newer cpus than the x86 32bit version.
There are x86_64 pc's that cannot run Minecraft Android x86_64, but x86.
x86 only needs SSSE3 support
x86_64 needs additionally SSE4.1, SSE4.2 and POPCNT support